### DISCUSSION TITLE
Movie support

### DISCUSSION THREAD

**Discussion by dr88dr88** [pymedusa/Medusa#9035] [Ideas & Feature requests]
I like to request to add also support for movie's. I use currently couch potato for movie download but this program is not receiving updates anymore. It would be great to have one integrated application for both tv shows and movies.

**Comment by PrzemekSkw** [comment]
I also will join Medusa Movies platform. I use Couchpotato and find it best. I try Radarr and Watcher but they don't work as good as Couchpotato or Medusa.

**Comment by p0psicles** [comment]
Yeah, would be cool. But a lot of work, little time. Would also help if experienced python or js devs would start contributing. But in a few years that has not happened. So this will probably also not happen

**Comment by nydave69** [comment]
Well I would still like to add my +1 to this..  

 It's really needed since Couchpotato has stopped updating..  I see they have pull requests buy users but the fixes never seem to make it to master.

**Comment by AhmedOsama** [comment]
Yes Adding movie support would be great +1 Please
